testing/tpm2-tools/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Alexander Sack <asac@pantacor.com>
+# Maintainer: Alexander Sack <asac@pantacor.com>
+pkgname="tpm2-tools"
+pkgver="4.0_rc2"
+pkgrel=0
+pkgdesc="TPM2 Tools"
+url="https://github.com/tpm2-software/tpm2-tools/"
+arch="all"
+license="BSD-2-Clause"
+makedepends="tpm2-tss-dev automake autoconf autoconf-archive libtool openssl-dev doxygen linux-headers curl-dev"
+subpackages="tpm2-tools-doc"
+sourcever="${pkgver/_/-}"
```

Comment:
Custom variables should start with a _ (so it should be _sourcever)

Reply:
done